test/e2e/framework/util.go

```diff
@@ -163,6 +163,9 @@ const (
 
 	// Number of times we want to retry Updates in case of conflict
 	UpdateRetries = 5
+
+	// number of objects that gc can delete in a second
+	gcThroughput = 15
```

Comment:
It came from previous settings - 10k pods -> 10 minutes, which gives 16.6 pods/s, so I rounded it down to 15:) I can do 10 and change the limit for 10k as well.